packages/kbn-security-hardening/console.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/* eslint-disable no-console */
+
+// From https://www.ascii-code.com/characters/control-characters,
+// but explicitly allowing the range \u0008-\u000F (line breaks, tabs, etc.)
+const CONTROL_CHAR_REGEXP = new RegExp('[\\u0000-\\u0007\\u0010-\\u001F]', 'g');
+
+export const unsafeConsole = {
+  // debug: console.debug.bind(console),
+  // error: console.error.bind(console),
+  info: console.info.bind(console),
+  log: console.log.bind(console),
```

Comment:
These are either not working as intended - when called from the console appender, the control chars are still being escaped via the code below.